.travis.yml

```diff
@@ -0,0 +1,17 @@
+after_script:
+  - docker images
+
+before_script:
+  - cd "$VERSION"
+  - IMAGE="litecoind:$VERSION"
+
+env:
+  - VERSION=0.10.4/alpine
```

Comment:
Since we're only going to maintain one version, I'd suggest removing this build matrix.